docs/can-guides/contribute/including-new-modules.md

```diff
@@ -0,0 +1,111 @@
+@page guides/contributing/include-new-modules Including New Modules
+@parent guides/contribute
+
+@description Learn how to add your module to this site.
+
+@body
+
+A project that is useful to the CanJS community can be included on the [CanJS documentation](http://canjs.com/) site.
+
+Before submitting a PR with these changes, make sure that the source module has [guides/contributing/documentation] and tests that follow [guides/contributing/code] standards set forth by the CanJS community.
+
+## Source Repo Documentation
+
+Create a markdown file in the `docs` folder of source repo and name it after the project.
```

Comment:
“source repo” isn’t defined before this section, so I would either define it in the intro (e.g. ‘In this guide, we’ll use the term “source repo” to refer to the repository of the package that you want to add to the ecosystem collection’) or make it more clear here (e.g. …‘folder in your package’s repository and’…).